app/assets/stylesheets/spina/conferences/primer_theme/application.sass

```diff
@@ -37,3 +37,10 @@
 
 address
   font-style: normal
+
+#form_frame
+  border: none
+  width: 100%
+  height: 90vh
+  @include breakpoint(lg)
+    height: 480px
```

Comment:
I would consider making this a uniform `70vh` because mobile users will need to be able to scroll the main page easily as well as the frame. On desktop I don't think it would hurt to have the frame a little taller. You could set a `max-height` if you want rather than using the breakpoint.

Reply:
sure, 70vh looks good to me.